pkg/authorization/authorizer/scope/converter.go

```diff
@@ -178,37 +179,37 @@ func (userEvaluator) Describe(scope string) (string, string, error) {
 	}
 }
 
-func (userEvaluator) ResolveRules(scope, namespace string, clusterPolicyGetter authorizationlister.ClusterPolicyLister) ([]authorizationapi.PolicyRule, error) {
+func (userEvaluator) ResolveRules(scope, namespace string, _ rbaclisters.ClusterRoleLister) ([]rbac.PolicyRule, error) {
 	switch scope {
 	case UserInfo:
-		return []authorizationapi.PolicyRule{
-			{Verbs: sets.NewString("get"), APIGroups: []string{userapi.GroupName, userapi.LegacyGroupName}, Resources: sets.NewString("users"), ResourceNames: sets.NewString("~")},
+		return []rbac.PolicyRule{
```

Comment:
I have the opposite feeling - I get really annoyed if I have to look up and down to figure out the context for a given line.  All of this is just subjective human preference, and I value consistency over personal preference (i.e. tools like `gofmt`).  We will have to agree to disagree on this, as I think we just have opposite preferences.